### ISSUE TITLE
Which fields do we really need for user data?

### ISSUE THREAD

**Issue by char23fj** [char23fj/361-Group20-Project#2]
At a bare minimum (and there's definitely merit in keeping it simple), we need:
-email
-zip code (or some location identifier)
-password

So here are some things to think about:
-Do we need to know people's names?  Are we possibly even better off not knowing (of course, we'll need names for billing information, but that's not a requirement up front)?
-Zip code alone probably suffices without adding another category for state, don't you think?
-email is necessary, user ID is not, but as a user of various sites I always appreciate the option of entering a short user ID rather than my email.  I could go either way.

**Comment by psigillito** [comment]
I agree we should keep the email, zip code, and password.  We probably wont touch the other things for the project so I dont mind if we get rid of them.  

The userId is a unique primary key for the table.  The user does not interact with it or know what it is.  It is used on the backend to identify users so I think it should stay.  Using an auto increment  id is easier to code with than using an email or first name & last name as a unique key.

**Comment by jarbeme** [comment]
I think that we should try to follow the UML diagram that we developed, even if we do not mess with them at least that way we can say a profile can be fully developed.
